Gemfile

```diff
@@ -26,7 +26,7 @@ group :development, :test do
   gem "faker"
   gem "i18n-tasks"
   gem "pry-rails"
-  gem "rspec-rails", "~> 3.4.2"
+  gem "fuubar", "~> 2.1.0.beta2"
```

Comment:
I'm in favour of removing this to help make RSpec changes in the future easier, but I know some other people have some different opinions :)